### PR TITLE
HTML heading tags to Markdown

### PR DESCRIPTION
**Pull request by jignesh-baldha**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will convert HTML heading tags to Markdown.

### CODE REVIEW

**Comment by shrielenee** [comment]
Thanks @jignesh-baldha! I added comments about a few changes to be complete before pushing back up for review. Also check out and resolve th emerge conflict listed. 🎉

**Comment by jignesh-baldha** [comment]
@shrielenee I've resolved conflict and did changes you said. Please review changes now.

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Blocks employ templates to generate {% glossarytooltip a2aff425-07dd-4bd6-9671-2
 
 To pass parameters use the <a href="#argument">`<argument></argument>`</a> instruction. 
 
-<h3 id="fedg_layout_xml-instruc_ex_cont">&lt;container&gt;</h3>
+### &lt;container&gt; {#fedg_layout_xml-instruc_ex_cont}
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]: Ideally, we would want these `&lt;` to be removed and replaced with the proper symbol for easier reading, but its not imperative for this PR. Keep this in mind for future revisions, however.

**Comment by shrielenee** [comment]
Your changes are approved @jignesh-baldha but I still see that there is a conflict. Can you fix that and push it back up for a final review and merging? Thanks! 😺

**Comment by jignesh-baldha** [comment]
@shrielenee Resolved conflict. Could you please review changes now.

**Pull request merged**